Print/src/RunSubrunEvent_module.cc

```diff
@@ -96,7 +96,7 @@ void mu2e::RunSubrunEvent::beginSubRun(art::SubRun const& subRun) {
   if (!_printSam) return;
 
   // add this subrun to the list, if it is not already there
-  unsigned long long id = subRun.run()*1000000 + subRun.subRun();
+  unsigned long long id = subRun.run()*1000000ull + subRun.subRun();
```

Comment:
Something to consider for a future PR.  Art has a art::SubRunID class.  Objects of that type are sortable and printable.  Did you consider using those instead of constructing your own identifier?   I think that your choice of a multiplier of 1E6 is safe but using art::SubRunID is explicitly safe.  It will be epsilon slower but I don' think that's a consider for this code.